### PR TITLE
Added jumpToPage option

### PR DESCRIPTION
**Pull request by patorjk**
This option, if true, adds a "Jump to Page" dropdown to the table footer that allows users to navigate to a specific page.

### CODE REVIEW

File: src/components/JumpToPage.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    textLabels,
+    rowsPerPage,
+    page,
+    changePage
+  } = props;
+
+  const textLabel = textLabels.pagination.jumpToPage;
+
+    let pages = [];
+  let lastPage = Math.min( 1000, getPageValue(count, rowsPerPage, 1000000));
```
Comment (wdh2100) [inline src/components/JumpToPage.js:63]: In the case of ServerSide, it seems that we need to find the value in another way.

For example, Add  `lastPage` props, options

File: src/components/JumpToPage.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {  
+  InputBase,  
+  MenuItem,  
+  Select,  
+  Toolbar,  
+  Typography
+} from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
```
Comment (wdh2100) [inline src/components/JumpToPage.js:10]: Method Higher-order component(`withStyles`)  looks better than method Hooks(`makeStyles`).

Especially in the case of functional component

`withStyles` to `makeStyles`

However, it is not an unconditional task.
It has nothing to do with behavior | logic.

File: src/components/JumpToPage.js (hunk 1 above)
Comment (patorjk) [reply src/components/JumpToPage.js:63]: It'll still work. When using serverSide, you're supposed to have the total count of the table (but you only display the current page of entries). This feature is actually ported over from a fork I was using and I have a bunch of serverSide tables.

File: src/components/JumpToPage.js (hunk 2 above)
Comment (patorjk) [reply src/components/JumpToPage.js:10]: Great catch! I first had the component as a class but then decided to make it a hook, however, I must have gotten distracted and forgot the withStyles/makeStyles part. Kind of interesting that everything still worked fine. I'll push an update here shortly though.

**Pull request merged**